### PR TITLE
Update to 1.15.0

### LINKED ISSUE #11: Update to latest version for python-ecmwf_grib
@kmuehlbauer we need to update here first. Are you doing it or do you want me to do it?

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://software.ecmwf.int/wiki/display/GRIB/GRIB+API+version+1.15.0+released for the changelog

Closes #11

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ about:
 
 extra:
     recipe-maintainers:
+        - kmuehlbauer
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Ping @kmuehlbauer :wink:

**Comment by kmuehlbauer** [comment]
Seems, the testfile-download failed.

**Comment by ocefpaf** [comment]
> Seems, the testfile-download failed.

Interment failures.

@kmuehlbauer I modified `build.sh` to use `cmake`. Can you take a quick look and merge if you are OK with the changes?

**Comment by kmuehlbauer** [comment]
@ocefpaf 

I'm not sure if we also need these lines here, but we will see, if it doesn't work:
https://github.com/conda-forge/eccodes-feedstock/blob/master/recipe/build.sh#L11

Unfortunately I have no merge-rights atm, but this is LGTM.

**Comment by ocefpaf** [comment]
> I'm not sure if we also need these lines here

The tests passed, but it does not hurt to add them back (I removed by accident).

> Unfortunately I have no merge-rights atm, but this is LGTM.

Yes, right. I need to merge this to actually add you :grimacing:

**Pull request merged**